src/components/SearchAndSort/SearchAndSort.js

```diff
@@ -725,28 +726,34 @@ export class SearchAndSort extends Component {
   };
 
   render() {
+    const { finishedResourceName } = this.props;
+
+    const isMappingProfile = finishedResourceName === ENTITY_KEYS.MAPPING_PROFILES;
     const source = this.getSource();
 
     return (
-      <Paneset>
-        <SRStatus ref={this.SRStatusRef} />
-        <Pane
-          id="pane-results"
-          defaultWidth="fill"
-          noOverflow
-          padContent={false}
-          renderHeader={this.renderPaneHeader}
-        >
-          <div className={css.paneBody}>
-            {this.renderSearch(source)}
-            <div className={css.searchResults}>
-              {this.renderSearchResults(source)}
+      <>
+        <Paneset>
+          <SRStatus ref={this.SRStatusRef} />
+          <Pane
+            id="pane-results"
+            defaultWidth="fill"
+            noOverflow
+            padContent={false}
+            renderHeader={this.renderPaneHeader}
+          >
+            <div className={css.paneBody}>
+              {this.renderSearch(source)}
+              <div className={css.searchResults}>
+                {this.renderSearchResults(source)}
+              </div>
             </div>
-          </div>
-        </Pane>
-        {this.renderDetailsPane(source)}
-        {this.renderCreateRecordLayer(source)}
-      </Paneset>
+          </Pane>
+          {!isMappingProfile && this.renderDetailsPane(source)}
+          {this.renderCreateRecordLayer(source)}
+        </Paneset>
+        {isMappingProfile && this.renderDetailsPane(source)}
```

Comment:
Look [there](https://github.com/folio-org/ui-data-import/pull/639/files#diff-7e8ccabec46feaba584bac318c32f358R475-R486) are props which `<FullScreenView>` component doesn't need, so consider splitting props for the full screen and for another one.